flexget/plugins/filter/translations.py

```
@@ -0,0 +1,594 @@
+            bool: is language
+        """
+
+        if isinstance(lang, Language):
+            return True
+
+        if not isinstance(lang, str) or lang in ['', 'und', UNKNOWN, NATIVE, DEFAULT, NONE, OTHER]:
+            return False
+
+        try:
+            mycode = Language.frommycode(lang)
+            if mycode:
+                lang = mycode.name
+        except babelfish.LanguageReverseError:
+            pass
+
+        try:
+            language = Language.fromietf(lang)
+        except (ValueError, babelfish.LanguageReverseError):
+            try:
+                language = Language.fromcode(lang, 'name')
+            except ValueError:
+                return False
+            except babelfish.LanguageReverseError:
+                return False
+
+        if isinstance(language, Language):
+            return True
+
+        return False
+
+    def _get_language(self, lang: Union[str, List[str], Language, List[Language]]) -> List[str]:
+        """Returns the language in text format
+
+        Args:
+            lang (Union[str, Language]): Language to return
+
+        Returns:
+            str: Language text
+        """
+
+        if not isinstance(lang, list):
+            languages = [lang]
+        else:
+            languages = lang
+
+        for key, lang in enumerate(languages):
+            if isinstance(lang, Language):
+                lang = lang.name
+
+            if not isinstance(lang, str) or lang in ['', 'und', UNKNOWN]:
+                languages[key] = UNKNOWN
+                continue
+
+            if lang is None or lang == NONE:
+                languages[key] = NONE
+                continue
+
+            if lang in [NATIVE, DEFAULT, OTHER]:
+                languages[key] = lang
+                continue
+
+            try:
+                mycode = Language.frommycode(lang)
+                if mycode:
+                    lang = mycode.name
+            except babelfish.LanguageReverseError:
+                pass
```

Will hide errors, is this always desirable? Log exception / debug instead?